src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js

```diff
@@ -62,18 +84,20 @@ const ItemAcquisition = ({ accordionId, itemId }) => {
 
         <Col xs={4}>
           <KeyValue
-            label={<FormattedMessage id="ui-inventory.acq.orderStatus" />}
-            value={order && <FormattedMessage id={`ui-inventory.acq.orderStatus.${order.workflowStatus}`} />}
+            label={<FormattedMessage id="ui-inventory.acq.acqMethod" />}
+            value={orderSetting?.value ?? <NoValue />}
```

Comment:
you don't need to provide NoValue as a default value here, KeyValue component has a check for it - https://github.com/folio-org/stripes-components/blob/master/lib/KeyValue/KeyValue.js#L24

Reply:
You are right, I removed `<NoValue />` everywhere.
Thank you!